cluster/addons/metadata-proxy/gce/metadata-proxy.yaml

```diff
@@ -37,6 +37,10 @@ spec:
       serviceAccountName: metadata-proxy
       hostNetwork: true
       dnsPolicy: Default
+      tolerations:
```

Comment:
since this is a critical addon should we give it:
```
  tolerations:
  - operator: "Exists"
    effect: "NoExecute"
  - operator: "Exists"
    effect: "NoSchedule"
```

Reply:
done

Reply:
in updating the operators, the network unavailable component got dropped... please reverify the structure of the toleration

Reply:
oh, I see... the intent was to tolerate any taint

Reply:
@liggitt , this suggests to tolerate everything.